### PR TITLE
fix Find and Schedule, finish list

### PR DESCRIPTION
**Pull request by yuyanglin28**

### CODE REVIEW

File: src/main/java/logic/parser/CheckCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import logic.command.CheckCommand;
+import logic.command.Command;
+import logic.command.HelpCommand;
+
+public class CheckCommandParser {
+    public static final String CHECK_USAGE = "'check' to check time crash.";
+
+    //@@author yuyanglin28
+    /**
+     * parse help command
```
Comment (chenyuheng) [inline src/main/java/logic/parser/CheckCommandParser.java]: wrong javadoc

File: src/main/java/logic/parser/list/ListTasksParser.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+public class ListTasksParser {
+
+    public static final String LIST_USAGE = "Usage: list tasks {all/todo} (picnum)";
+    private static final Pattern BASIC_LIST_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
+    private static final String ALL = "all";
+    private static final String TODO = "todo";
+    public static final String PICNUM = "picnum";
+
+    /**
+     * Parses add commands.
```
Comment (chenyuheng) [inline src/main/java/logic/parser/list/ListTasksParser.java]: wrong javadoc

File: src/main/java/logic/parser/list/ListTasksTodoParser.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+import common.DukeException;
+import logic.command.Command;
+import logic.command.list.ListTasksTodoCommand;
+import logic.command.list.ListTasksTodoPicNumCommand;
+
+public class ListTasksTodoParser {
+
+    /**
+     * Parses add commands.
```
Comment (chenyuheng) [inline src/main/java/logic/parser/list/ListTasksTodoParser.java]: also

Comment (yuyanglin28) [reply src/main/java/logic/parser/list/ListTasksTodoParser.java]: change it later

**Pull request merged**